### DISCUSSION TITLE
Scaling options for Stickers

### DISCUSSION THREAD

**Discussion by nisaeggu** [silveredgold/beta-protection#23] [Ideas]
**Issue:** The stickers are only as big as the area that is being censored, which makes some stickers hard to work with. They either disappear or are super small to the point where they disappear.

**Idea:** Make a true or false option where you can set either a **fixed size** or **scale to censorbox**. If possible it would be great if it adjust automatically a little larger than the censored area with a slider to adjust it.

**Comment by silveredgold** [comment]
Do you have the censoring level turned down? The size of the stickers is controlled not just by the size of your sticker, but also the censoring level chosen on whatever body part you have enabled stickers for. If you set it to a higher value, it will cover a larger area.
